Multisig/Data/Services/Safe Client Gateway Service/ProposeTransactionRequest.swift

```diff
@@ -9,8 +9,8 @@
 import Foundation
 
 struct ProposeTransactionRequest: JSONRequest {
-    let safe: AddressString
-    let sender: AddressString
```

Comment:
I think AddressString is more restrictive and thus a bit better. Why did you decide to change it to Address/String? (And why not Address/Address then?)